### PR TITLE
VR-6560 repository public within org

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

**Review by nhatsmrt — APPROVED** [review]
Looks solid to me, based on my limited experience with writing tests for public_within_org. Defer to @convoliution for more in-depth review.

File: client/verta/verta/client.py [hunk 1]
```diff
@@ -557,7 +557,7 @@ def set_experiment_run(self, name=None, desc=None, tags=None, attrs=None, id=Non
 
         return self._ctx.expt_run
 
-    def get_or_create_repository(self, name=None, workspace=None, id=None):
+    def get_or_create_repository(self, name=None, workspace=None, id=None, public_within_org=None):
```
Comment (convoliution) [inline client/verta/verta/client.py:560]: @evgeniilezhnin Could you also add `public_within_org` to the docstring's **Parameters** section? Should be able to copy&paste from `set_project()`.

**Review by convoliution — APPROVED** [review]
Makes sense to me. One request for docstring.

File: client/verta/verta/client.py (hunk 1 above)
Comment (evgeniilezhnin) [reply client/verta/verta/client.py:560]: yes, added

**Pull request merged**